content/03.results.md

```diff
@@ -232,15 +232,15 @@ The mutual exclusivity of signatures 3 and MMR2 corroborates a previous report s
-| PT_3CHB9PK5               | BS_8AY2GM4G               | 7316-2085 | Progressive       | Solid Tissue      | Unknown                                          | CMMRD                 | NM_000179.3(MSH6):c.3439-2A>G (LP)            | 321.6 | HGG, H3 wildtype, TP53 loss        |
-| PT_EB0D3BXG               | BS_F0GNWEJJ               | 7316-3311 | Progressive       | Solid Tissue      | Nivolumab, Radiation                             | None documented       | None detected                                 | 26.3  | Metastatic NBL, MYCN non-amplified |
-| PT_JNEV57VK               | BS_85Q5P8GF               | 7316-2594 | Initial CNS Tumor | Solid Tissue      | Temozolomide, Radiation                          | Lynch Syndrome        | NM_000251.3(MSH2):c.1906G>C (p.Ala636Pro) (P) | 4.7   | DMG, H3 K28, TP53 loss             |
-| PT_JNEV57VK               | BS_HM5GFJN8               | 7316-3058 | Progressive       | Derived Cell Line | Nivolumab                                        | Lynch Syndrome        | NM_000251.3(MSH2):c.1906G>C (p.Ala636Pro) (P) | 35.9  | DMG, H3 K28, TP53 loss             |
-| PT_JNEV57VK               | BS_QWM9BPDY               | 7316-3058 | Progressive       | Derived Cell Line | Nivolumab                                        | Lynch Syndrome        | NM_000251.3(MSH2):c.1906G>C (p.Ala636Pro) (P) | 7.4   | DMG, H3 K28, TP53 loss             |
-| PT_JNEV57VK               | BS_P0QJ1QAH               | 7316-3058 | Progressive       | Solid Tissue      | Nivolumab                                        | Lynch Syndrome        | NM_000251.3(MSH2):c.1906G>C (p.Ala636Pro) (P) | 6.3   | DMG, H3 K28, TP53 activated        |
-| PT_S0Q27J13               | BS_P3PF53V8               | 7316-2307 | Initial CNS Tumor | Solid Tissue      | Temozolomide, Irinotecan, Radiation              | None documented       | None detected                                 | 15.5  | HGG, H3 wildtype, TP53 activated   |
+| PT_0SPKM4S8               | BS_VW4XN9Y7               | 7316-2640 | Initial CNS Tumor | Solid Tissue      | Radiation, Temozolomide, CCNU                    | None documented       | NM_000535.7(PMS2):c.137G>T (p.Ser46Ile) (LP)  | 187.4 | HGG, H3 wildtype, TP53 activated   |
+| PT_3CHB9PK5               | BS_20TBZG09               | 7316-515  | Initial CNS Tumor | Solid Tissue      | Radiation, Temozolomide, Irinotecan, Bevacizumab | CMMRD                 | NM_000179.3(MSH6):c.3439-2A>G (LP)            | 307   | HGG, H3 wildtype, TP53 loss        |
+| PT_3CHB9PK5               | BS_8AY2GM4G               | 7316-2085 | Progressive       | Solid Tissue      | Radiation, Temozolomide, Irinotecan, Bevacizumab | CMMRD                 | NM_000179.3(MSH6):c.3439-2A>G (LP)            | 321.6 | HGG, H3 wildtype, TP53 loss        |
```

Comment:
When did this become known?

Reply:
This is a matter of including the Initial CNS Tumor treatments as part of the Progressive BSID treatments from the same participant in this table per 

> Also, do you think it will be clear to readers that for the progressive samples that the prior therapy (after the initial sample) would also have included radiation therapy?